contracts/TwabRewards.sol

```diff
@@ -152,8 +157,13 @@ contract TwabRewards is ITwabRewards {
         uint8 _extendedNumberOfEpochs = _promotion.numberOfEpochs + _numberOfEpochs;
         _promotions[_promotionId].numberOfEpochs = _extendedNumberOfEpochs;
 
-        uint256 _amount = _numberOfEpochs * _promotion.tokensPerEpoch;
-        _promotion.token.safeTransferFrom(msg.sender, address(this), _amount);
+        unchecked {
```

Comment:
I don't like that this whole block is unchecked.  Can we target only the arithmetic?

Reply:
Done: https://github.com/pooltogether/v4-periphery/pull/41/commits/d7ac3dc31582616866c2b55d0d94f7baade340f1